README.md

```
@@ -14,50 +14,56 @@ _Organize ideas and collaborate using Markdown, a lightweight language for text
+## This is an `<h2>` header
+
+###### This is an `<h6>` header, which is the smallest
+```
+
+#### How it looks
+
+# This is an `<h1>` header, which is the largest
+
+## This is an `<h2>` header
+
+###### This is an `<h6>` header, which is the smallest
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h6

(MD001, heading-increment)